src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -250,13 +250,13 @@ private void triggerPr(final GhprbTrigger trigger,
             public void run() {
                 try {
                     trigger.handlePR(pr);
-                } catch (Exception e) {
+                } catch (Throwable th) {
```

Comment:
In general, I've been told catching Throwable is a bad idea.  Can a more seasoned Jenkins dev review this PR?

cc @jenkinsci/code-reviewers

Reply:
If the intention here is to catch `Exception` and `Error`, then check for those types and if it's not either of those types then rethrow throwable.  I'm just concerned that we're swallowing all throwable.

Reply:
Also a quote from the `java.lang.Error` javadoc:

> An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch.

I need an adult 🆘

Reply:
It's a bad style for sure. But, you know, throwing it from the code is a bad idea as well :(

Reply:
In practice the main things it is problematic to catch are `OutOfMemoryError` (generally no practical recovery, so does not make much difference either way); and `ThreadDeath` (you need to be sure to rethrow quickly at least).

Reply:
I'll open up a followup PR @jglick.  I guess I merged too quickly.